src/Fantomas/Context.fs

```diff
@@ -599,6 +599,21 @@ let internal colPreEx f2 f1 (c: seq<'T>) f (ctx: Context) =
     else
         colEx f1 c f (f2 ctx)
 
+/// Similar to col, but apply two more functions fStart, fEnd at the beginning and the end if the input sequence is bigger thn one item
+let internal colSurr fStart fEnd f1 (c: list<'T>) f (ctx: Context) =
```

Comment:
I was a bit unsure about this abstraction at first.
I'm not entirely convinced `col f1 c f` should be part of this abstraction.
At the same time, I have no example code where a more general abstraction would make sense, so let's just go with this.